dali/pipeline/operators/resize/resize_base.h

```diff
@@ -60,6 +62,19 @@ class DLL_PUBLIC ResizeBase : public ResamplingFilterAttr {
     kernels::KernelContext context;
     kernels::KernelRequirements requirements;
     kernels::ScratchpadAllocator scratch_alloc;
+    unique_ptr<void, void(*)(void*)> instance = { nullptr, free };
```

Comment:
```suggestion
    std::unique_ptr<void, void(*)(void*)> instance = { nullptr, free };
```
since you're using `std` before